src/medCore/process/medAbstractProcess.h

```diff
@@ -32,6 +32,8 @@ class MEDCORE_EXPORT medAbstractProcess : public dtkAbstractProcess
     medAbstractProcess( medAbstractProcess * parent = NULL );
     virtual ~medAbstractProcess();
 
+    using dtkAbstractProcess::setInput;
```

Comment:
What I mean is that the method dtkAbstractProcess::setInput is not used by medAbstractProcess that's why it redefined as private. It can be redefined as public  with a covariant type of dtkAbstractData* as argument in an implementation of medAbstractProcess (maybe it is in your particular case) but it's just a matter of luck.